lib/ProMotion/screen/screen_module.rb

```diff
@@ -70,6 +70,13 @@ def view_did_disappear(animated)
     def on_disappear; end
     def on_dismiss; end
 
+    def did_receive_memory_warning
+      self.on_memory_warning
+    end
+    def on_memory_warning
+      mp "Received memory warning in #{self.inspect}. You should implement on_memory_warning in your secreen.", force_color: :red
```

Comment:
I kind of like it. Lets people know that we're human and not gem generating robots.